example/android/app/build.gradle

```diff
@@ -135,13 +158,30 @@ android {
             }
         }
     }
+
+    packagingOptions {
+        pickFirst '**/armeabi-v7a/libc++_shared.so'
+        pickFirst '**/x86/libc++_shared.so'
+        pickFirst '**/arm64-v8a/libc++_shared.so'
+        pickFirst '**/x86_64/libc++_shared.so'
+        pickFirst '**/x86/libjsc.so'
+        pickFirst '**/armeabi-v7a/libjsc.so'
+    }
 }
 
 dependencies {
-    implementation project(':react-native-share')
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation project(':react-native-share')
+    implementation "androidx.appcompat:appcompat:${rootProject.ext.supportLibVersion}"
```

Comment:
This is already a dependency of `react-native`. You can remove the reference here (leave `app/build.gradle` untouched).

Reply:
nice - done